cfgagent/intfconf.cpp

```diff
@@ -0,0 +1,126 @@
+#include <string.h>
+#include "logger.h"
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "tokenize.h"
+#include "ipprefix.h"
+#include "intfconf.h"
+#include "exec.h"
+
+using namespace std;
+using namespace swss;
+
+#define VLAN_PREFIX         "Vlan"
+#define LAG_PREFIX          "PortChannel"
+
+IntfConf::IntfConf(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, vector<string> tableNames) :
```

Comment:
as requested above, can you rename to IntfMgr

Reply:
So process and binary name will be changed to VlanMgr/IntfMgr/SwitchMgr,  you want to change the internal class name to xxxMgr too?   IntfConfd/VlanConfd were chosen per the offline meeting discussion.  Yes, class name may be changed.